test/graph_test/CMakeLists.txt

```diff
@@ -10,7 +10,7 @@ target_include_directories(
         ../include/
 )
 
-target_link_libraries(graph_test PUBLIC GTest::GTest kuzu)
+target_link_libraries(graph_test PUBLIC KUZU_GTEST kuzu)
```

Comment:
Optional, KUZU_GTEST -> GTEST_LIB? GTEST doesn't really have any connection to KUZU.